server/src/modules/authors/author.entity.ts

```diff
@@ -12,23 +12,23 @@ import { CommonEntity } from 'src/common/entities';
 
 @Entity({ name: 'authors' })
 export class AuthorEntity extends CommonEntity {
-  @ApiProperty()
+  @ApiProperty({ type: 'string', format: 'uuid' })
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
-  @ApiProperty({ maxLength: 32 })
+  @ApiProperty({ type: 'string', maxLength: 32, uniqueItems: true })
   @Column({ length: 32, unique: true })
   name: string;
 
   @ApiHideProperty()
   @OneToMany(() => TrackEntity, ({ author }) => author, { nullable: true })
   tracks: TrackEntity[];
 
-  @ApiProperty({ readOnly: true })
+  @ApiProperty({ type: 'string', readOnly: true, format: 'date-time' })
```

Comment:
It would be better not to assing type of createdAt and updatedAt as string, will transform it to type Date